docs/examples/101/sql-database/main.bicep

```diff
@@ -0,0 +1,47 @@
+param serverName string {
+  metadata: {
+    description: 'The name of the SQL logical server.'
+  }
+  default: uniqueString('sql', resourceGroup().id)
+}
+param sqlDBName string {
+  metadata: {
+    description: 'The name of the SQL Database.'
+  }
+  default: 'SampleDB'
+}
+param location string {
+  metadata: {
+    description: 'Location for all resources.'
+  }
+  default: resourceGroup().location
+}
+param administratorLogin string {
+  metadata: {
+    description: 'The administrator username of the SQL logical server.'
+  }
+}
+param administratorLoginPassword string {
+  metadata: {
+    description: 'The administrator password of the SQL logical server.'
+  }
+  secure: true
+}
+
+resource serverName_resource 'Microsoft.Sql/servers@2019-06-01-preview' = {
```

Comment:
out of curiosity, are these resource identifiers from the decompiler? `_resource` feels redundant

Reply:
Yes, I'm trying to learn how to build bicep files so the decompiler is helpful ;)